tests/test_autotuner.py

```diff
@@ -50,9 +50,10 @@ def test_at_is_actually_working(shape, expected):
 def test_mode_runtime_forward():
     """Test autotuning in runtime mode."""
     grid = Grid(shape=(96, 96, 96))
+    g = Function(name='g', grid=grid)
     f = TimeFunction(name='f', grid=grid)
 
-    op = Operator(Eq(f.forward, f + 1.), openmp=False)
+    op = Operator(Eq(f.forward, f + g + 1.), openmp=False)
```

Comment:
Is there an issue with the previous setup?

Reply:
no just the new blocking heuristic (don't block if there's definitely no data locality to exploit) that need to be bypassed somehow

Reply:
Exactly. But did you **sanity check** for some bigbig problem that indeed we are fine?

Reply:
I genuiinely have no idea what you're talking about?

Reply:
I mean whether we checked that our hypothesis that no data reuse exists is 100% true

Reply:
Still don't follow

Look at the code (symbolics or C, doesn't matter)

Where do you see data locality? What Function would induce data locality here in this test (or the others which have changed)?

Reply:
Totally agree. Just saying it could be easy to cross-verify